### ISSUE TITLE
Unable add nano scroll on body element?

### ISSUE THREAD

**Issue by Ram-SR** [jamesflorentino/nanoScrollerJS#335]
I want ro add nano scroll on body element. But as per the docs, the structure should be 
`<div class="nano"><div class="nano-content"></div></div>`

then how to add nano scroll on body element?

**Comment by kristerkari** [comment]
I don't think that having nanoscroller on `body` element is supported, the library is meant to be used with `div` elements. You can of course try to see if it happens to work with `body` + a `div` element.
